crates/web-sys/Cargo.toml

```diff
@@ -22,3 +22,1156 @@ js-sys = { path = '../js-sys', version = '0.2.4' }
 futures = "0.1"
 wasm-bindgen-test = { path = '../test', version = '0.2.19' }
 wasm-bindgen-futures = { path = '../futures', version = '0.2.19' }
+
+# This list is generated by passing `__WASM_BINDGEN_DUMP_FEATURES=foo` when
+# compiling this crate which dumps the total list of features to a file called
+# `foo`.
+#
+# Each one of these features activates the corresponding type, allowing bindings
+# to be generated for it. Note that we may eventually add "groupings" of
+# features to enable a convenient set of features all at once. For now, though,
+# the features must all be manually activated.
+[features]
+AbortController = []
+AbortSignal = []
```

Comment:
Yeeeeeaaaaaaah!

Reply:
Oh also: this comment about features should probably go into the README and/or src/lib.rs too